atc/exec/put_step_test.go

```diff
@@ -371,10 +381,30 @@ var _ = Describe("PutStep", func() {
 						{
 							Artifact:        volume1,
 							DestinationPath: "/tmp/build/put/input1",
+							FromCache:       false,
 						},
 						{
 							Artifact:        volume2,
 							DestinationPath: "/tmp/build/put/input2",
+							FromCache:       true,
+						},
+					}))
+				})
+			})
+
+			Context("when only some inputs are from cache ", func() {
+				BeforeEach(func() {
+					putPlan.Inputs = &atc.InputsConfig{
+						Specified: []string{"input2"},
+					}
+				})
+
+				It("runs with specified inputs", func() {
```

Comment:
How is a specified input related to cache?

Reply:
I think this is a typo. Goal of this test is to make sure when only input is from cache, container spec is well populated, `FromCache` should be true.